subsys/bluetooth/shell/bt.c

```diff
@@ -904,7 +904,7 @@ static int cmd_scan(const struct shell *shell, size_t argc, char *argv[])
 		} else if (!strcmp(arg, "nodups")) {
 			options &= ~BT_LE_SCAN_OPT_FILTER_DUPLICATE;
 		} else if (!strcmp(arg, "wl")) {
-			options |= BT_LE_SCAN_OPT_FILTER_WHITELIST;
+			options |= BT_LE_SCAN_OPT_FILTER_ACCEPT_LIST;
```

Comment:
I guess we don't count the shell as public API, right? Otherwise this should also be updated.

Reply:
Right, there is another PR coming for the internal implementation. These changes here are present to eliminate any warnings related to using deprecated macros.